frontend/components/site/SiteScans.jsx

```diff
@@ -71,21 +72,23 @@ function SiteScans() {
               <h3>Welcome to the Site Scans beta!</h3>
               <p>
                 {/* eslint-disable-next-line max-len */}
-                Pages is now offering automated monthly quality scans of customer production sites. These scans examine your Pages site
-                <sup>*</sup>
-                {' '}
-                {/* eslint-disable-next-line max-len */}
-                for common website issues and provide guidance and resources for remediation. Site Scans are part of a larger effort to help our customers comply with OMB Memo 23-22 and ATO requirements.
+                Pages is now offering automated monthly quality scans of customer production sites. These scans examine your Pages site for common website issues and provide guidance and resources for remediation. Site Scans are part of a larger effort to help our customers comply with OMB Memo 23-22 and ATO requirements.
               </p>
               { siteBuildTasks.length > 0 && (
                 <>
-                  <p>Your next automated monthly scans for this site are scheduled for:</p>
+                  <p>
+                    The next automated monthly scans for the
+                    <b>{branchToBeScanned}</b>
```

Comment:
Needs a space, but maybe we should look into use more string literals to make this easier